### ISSUE TITLE
Accessibility: The "Dropdown menu" control is marked up as a <label>

### ISSUE THREAD

**Issue by kemiga** [sky-uk/toolkit-ui#149]
## Expected Behavior
Interactive controls should be marked up as interactive elements in order to convey their purpose programmatically. This also ensures that they receive keyboard focus and that screen readers announce them to users as interactive.

## Current Behavior
Currently, the "Dropdown menu" is marked up as a <label> element. This means that it doesn't receive keyboard focus and is announced by the screen reader as a "text element" instead of an interactive element.

## Suggestion
The dropdown menu should ideally be marked up as a <select> element. However, if this is not possible, it could be marked up as a <button> with aria attributes, e.g. aria-controls, to associate it with the content it is hiding and showing.

**Comment by joebell93** [comment]
@kemiga we're planning to deprecate dropdown menu in the near future to be handled via JS rather than a checkbox hack

**Comment by joebell93** [comment]
Fixed via https://github.com/sky-uk/toolkit-ui/pull/161
